src/conf/startup.properties

```diff
@@ -43,16 +43,22 @@
                         org.apache.falcon.workflow.WorkflowJobEndNotificationService, \
                         org.apache.falcon.service.ProcessSubscriberService,\
                         org.apache.falcon.service.FalconJPAService,\
-                        org.apache.falcon.extensions.ExtensionService,\
                         org.apache.falcon.service.LifecyclePolicyMap,\
                         org.apache.falcon.entity.store.ConfigurationStore,\
                         org.apache.falcon.rerun.service.RetryService,\
                         org.apache.falcon.rerun.service.LateRunService,\
-                        org.apache.falcon.metadata.MetadataMappingService,\
                         org.apache.falcon.service.LogCleanupService,\
                         org.apache.falcon.service.GroupsService,\
                         org.apache.falcon.service.ProxyUserService
 
+##Add if you want to use Meatadata Service
+## Also enable all the properties which starts with falcon.graph prefix
+#org.apache.falcon.metadata.MetadataMappingService,\
+
+##Add if you want to use Trusted and NonTrusted Extensions
+## In case of distributed Mode enable ExtensionService only on Prism
+#org.apache.falcon.extensions.ExtensionService,\
```

Comment:
Call it Custom or User extensions.  Is it ok if this Service is at the bottom of the list? If not, call out where it needs to appear.